melodic/distribution.yaml

```diff
@@ -32,6 +31,11 @@ repositories:
       type: git
       url: https://github.com/ros-industrial/abb_driver.git
       version: kinetic-devel
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
Releasing from the Kinetic release branch as there is no difference between the Kinetic and Melodic versions.

Reply:
Hm, I don't know if that actually works.  The reason being that the "overlays" that we use in the release repository also contain the name of the Ubuntu/Debian distribution that they are being released on, and that is different between Kinetic and Melodic.  Certainly I haven't seen this use before; @nuclearsandwich , will this work?

Reply:
I can create a `melodic` release branch. It'll just be a copy really.

Reply:
Done.